tests_and_analysis/test/legacy_tests/test_interpolation_data.py

```diff
@@ -69,7 +69,7 @@ def setUp(self):
                                              [0, 0, 1]])
         self.expctd_data = expctd_data
 
-        self.path = os.path.join(get_data_path(), 'interpolation', 'LZO')
+        self.path = os.path.join(get_data_path(), 'force_constants', 'LZO')
```

Comment:
Now the testing has been implemented elsewhere, we can probably remove this old testing file altogether. We can also remove a lot of the files that this test was using (`lzo_unique_sc_offsets.txt`, `lzo_unique_sc_i.txt` etc.)

Reply:
Removed, I think I have removed all the test files as well